iceoryx_utils/include/iceoryx_utils/posix_wrapper/semaphore.hpp

```diff
@@ -157,8 +157,8 @@ class Semaphore : public DesignPattern::Creation<Semaphore, SemaphoreError>
     bool m_isNamedSemaphore = true;
     bool m_isShared = false;
 
-    mutable sem_t m_handle;
-    mutable iox::relative_ptr<sem_t> m_handlePtr = &m_handle;
+    mutable iox_sem_t m_handle;
+    mutable iox::relative_ptr<iox_sem_t> m_handlePtr = &m_handle;
 
   private:
```

Comment:
private already on line 153 declared

Reply:
I like to write it twice, once for all the private members and another time for all the private methods. Its my artistic freedom ;)